src/lib/support/logging/CHIPLogging.cpp

```diff
@@ -88,6 +88,7 @@ static const char ModuleNames[] = "-\0\0" // None
                                   "SH\0"  // Shell
                                   "DL\0"  // DeviceLayer
                                   "SPL"   // SetupPayload
+                                  "APP"   // App
```

Comment:
Do we want separate names for "App" in the sense of "the CHIP-provided part of the app, but conceptually at the app layer" and "App" in the sense of "the part of the app that is really app-specific"?  Followup OK here; would just involve evaluating all the uses of the `App` log module...

Reply:
I guess "APP" should be used for CHIP provided application layer libraries.

Reply:
I can't come up with another name for logging in example apps now, one solution is just using `NotSpecified`. Will this be ok?

Reply:
Reviewed usage of logs, changed logging in AppTask.cpp back to `NRF_LOG`, logs in data models uses `Zcl`, and logs in Server.cpp will use `AppServer` since this part should be merged into CHIP core library in the future IMHO.